python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -255,7 +255,7 @@ def cache(
     with open(arg_data_file, "rb") as fh:
         arg_data = cPickle.load(fh)
 
-    sys.path = arg_data["sys_path"]
+    sys.path.extend(arg_data["sys_path"])
```

Comment:
+1 for prepend. I think that's how we have approached in in other places. If toolkit requires something, we should make sure that we get exactly the version of which we expect. Say for example someone has an older version of the python API already in the pythonpath - extending the path would cause errors in toolkit (hard to track down) as it is expecting a more recent version of the API.